modules/face/samples/sampleKazemi.cpp

```diff
@@ -0,0 +1,106 @@
+#include<iostream>
+#include<vector>
+#include<fstream>
+#include "opencv2/face/face_alignment.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgcodecs.hpp"
+using namespace std;
+using namespace cv;
+int main(int argc,char** argv){
+    //Give the file containing the filenames of filenames as argument
+    if(argc!=2){
+        cout<<"No file found";
+        return 0;
+    }
+    string file = argv[1];
```

Comment:
Again thats just me, but white lines tend to give structure between different parts